test/videojs-http-streaming.test.js

```diff
@@ -3654,28 +3654,28 @@ QUnit.test('cleans up the buffer when loading VOD segments', function(assert) {
     request: this.requests[2],
     mediaSource: mpc.mediaSource,
     segmentLoader: mpc.mainSegmentLoader_,
-    clock: this.clock,
-    tickClock: false
+    clock: this.clock
   }).then(() => {
-
-    // the seek will have removed everything to the duration of the video, so we want to
-    // only start tracking removes after the seek, once the next segment request is made
-    this.player.currentTime(120);
```

Comment:
This operation used to be synchronously call `masterPlaylistController.setCurrentTime` due to how middleware works. Now it doesn't so the test had to be reworked somewhat to account for that.